README.md

```diff
@@ -11,44 +11,46 @@ This repo contains two main programs:
 
 ## Installation
 
-pip install git+https://github.com/trompamusic/trompa-ce-client.git@tpl
+    pip install git+https://github.com/trompamusic/trompa-ce-client.git
 
-git clone https://github.com/trompamusic/tpl
+    git clone https://github.com/trompamusic/tpl
 
-cd tpl
-
-pip install -r requirements.txt
-
-python setup.py install
```

Comment:
we should decide if these instructions are for development, or setting up and running the TPL in a specific location, and update the instructions as required (#6)